source/docs/software/dashboards/dashboard-intro.rst

```diff
@@ -0,0 +1,40 @@
+
+:ref:`LabVIEW Dashboard <docs/software/dashboards/labview-dashboard/index:LabVIEW Dashboard>` (Driver / Programming) - easy to use and provides a lot of features straight out of the box like: camera streams, autonomous selection, and joystick feedback.  It can be customized using LabVIEW by creating a new Dashboard project.  While it :ref:`can be used <docs/software/dashboards/labview-dashboard/using-the-labview-dashboard-with-c++-java-code:Using the LabVIEW Dashboard with C++/Java Code>` by Java or C++ teams, they generally prefer SmartDashboard or Shuffleboard which can be customized in their respective language.
+
+:ref:`SmartDashboard <docs/software/dashboards/smartdashboard/index:SmartDashboard>` (Driver) - simple and efficient dashboard that uses relatively few computer resources.  It does not have the fancy look or some of the features Shuffleboard has, but it displays network tables data with a variety of widgets without bogging down the driver station computer.
+
+:ref:`Shuffleboard <docs/software/dashboards/shuffleboard/index:Shuffleboard>` (Driver) - straightforward and easily customizable dashboard. It displays network tables data using a variety of widgets that can be positioned and controlled with robot code. It includes many extra features like: tabs, recording / playback, and advanced custom widgets.
+
+:ref:`Glass <docs/software/dashboards/glass/index:Glass>` (Programming) - robot data visualization tool. Its GUI is extremely similar to that of the :ref:`Simulation GUI <docs/software/wpilib-tools/robot-simulation/simulation-gui:Simulation Specific User Interface Elements>`. In its current state, it is meant to be used as a programmer's tool rather than a proper dashboard in a competition environment.
+
+:ref:`AdvantageScope <docs/software/dashboards/glass/index:Glass>` (Programming) - robot diagnostics, log review/analysis, and data visualization application.  It reads logs in WPILOG, DS log, and RLOG file formats, plus live robot data viewing using NT4 or RLOG streaming.
+
+Third Party Dashboards
+----------------------
+
+`FRC Web Components <https://github.com/frc-web-components/frc-web-components>`__ (Driver) - A web-based dashboard that can be installed as a standalone application, or as a javascript package for custom dashboard solutions.
+
+`Elastic <https://github.com/Gold872/elastic-dashboard>`__ (Driver) - simple and modern Shuffleboard alternative made by Team 353. It is meant to serve as a dashboard for competition but can also be used for testing.  Featuring draggable and resizable card widgets and automatic IP retrieval from the FRC Driver Station
+
+`QFRCDashboard <https://github.com/binex-dsk/QFRCDashboard>`__ (Driver) - reliable, high-performance, low-footprint dashboard.  QFRCDashboard has been specifically designed to use up as little resources as possible.
```

Comment:
Reliable might overselling it, since it was just released and hasn't had much use.

Reply:
I copied that from their page.  Not sure how to resolve that.  Maybe say like "described as" or something.  Reliability is in the eye of the beholder, but I agree that it is near impossible to make that claim with as little testing as it has likely had when compared with say shuffleboard.

Reply:
I didn't think the others needed that label so just added that one